sycl/include/CL/sycl/detail/pi.h

```diff
@@ -531,6 +531,7 @@ constexpr pi_queue_properties PI_QUEUE_PROFILING_ENABLE =
 constexpr pi_queue_properties PI_QUEUE_ON_DEVICE = CL_QUEUE_ON_DEVICE;
 constexpr pi_queue_properties PI_QUEUE_ON_DEVICE_DEFAULT =
     CL_QUEUE_ON_DEVICE_DEFAULT;
+constexpr pi_queue_properties PI_QUEUE_IN_ORDER_EXEC_MODE_ENABLE = 1 << 4;
```

Comment:
Why do you need this? I though that if there were no `PI_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE` specified means in-order is requested.

Reply:
that could be done too. will change this.